src/tests/components/Auth/Logo/index.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import { render } from '@testing-library/react';
+
+import LogoWithLabel from 'components/Auth/Logo';
+import logo from 'images/logo_white.png';
+
+describe('When LogoWithLabel component is mounted', () => {
+  const label = 'Sign in to Nimble';
+
+  it('renders button with submit type', () => {
+    const { getByTestId } = render(<LogoWithLabel label={ label } />);
+    const logoImage = expect(getByTestId('logo-test'));
```

Comment:
How about `getByAltText` instead of `getByTestId` here ?

Reply:
Changed!